package.json

```diff
@@ -30,13 +30,17 @@
     "@types/bcryptjs": "^2.4.2",
     "@types/redis": "^2.8.28",
     "@types/redis-mock": "^0.17.0",
-    "axios": "^0.21.0",
+    "@types/request-promise": "^4.1.47",
+    "axios": "^0.21.1",
     "bcryptjs": "^2.4.3",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "firebase-admin": "^9.4.2",
+    "firebase-mock": "^2.3.2",
     "joi": "^17.3.0",
     "redis": "^3.0.2",
-    "redis-mock": "^0.56.3"
+    "redis-mock": "^0.56.3",
+    "request-promise": "^4.2.6"
```

Comment:
I'm seeing some unmet peer dependency messages, I think all you need to do is install these dependencies and the messages will go away

![image](https://user-images.githubusercontent.com/49849754/108642975-3a127200-745d-11eb-91f6-65d41a757154.png)